lang/en.json

```diff
@@ -644,6 +644,10 @@
   "PAD_SOURCE_MISSING": "Selected PAD source '{{sourceName}}' is missing",
   "VALID_FROM": "Valid From",
   "PAUSED": "Paused",
+  "CAPACITY_START": "Start CU address",
+  "CAPACITY_START_NOT_USED": "Automatically set by mux",
+  "H_USE_CAPACITY_START": "Start CU address for all services of this provider",
```

Comment:
Well, this may be difficult to understand since no one wants all their services to start at the same CU :)

Isn't it more someting like
"Start CU of the range allocated for this provider."

Reply:
Yeah, I changed this wording too many times and got blind of what the end result actually says. I'll use your proposal and merge.